include/libdnf/repo/repo.hpp

```diff
@@ -391,7 +391,11 @@ class Repo {
 
     void internalize();
 
-    void reset_metadata_expired();
+    /// If the repository is not already marked as expired, it checks for the presence of the storage cache expiration
```

Comment:
What about change `storage` to `stored`?

Reply:
I changed "storage" to "repository".